platform-sdk/swirlds-platform-core/src/jmh/java/com/swirlds/platform/core/jmh/EventSerialization.java

```diff
@@ -78,4 +88,17 @@ public void serializeDeserialize(final Blackhole bh) throws IOException {
         outStream.writeSerializable(event, false);
         bh.consume(inStream.readSerializable(false, PlatformEvent::new));
     }
+
+    @Benchmark
+    @BenchmarkMode(Mode.Throughput)
+    @OutputTimeUnit(TimeUnit.MILLISECONDS)
+    public void hashingBase(final Blackhole bh) throws IOException {
+        // results on Timo's M3 Max MacBook Pro:
+        //
+        // Benchmark                       (seed)   Mode  Cnt     Score     Error   Units
+        // EventSerialization.hashingBase       0  thrpt    3  1680.478 ± 173.379  ops/ms
+
+        event.serializeLegacyHashBytes(outputStream);
```

Comment:
Any output should be consumed by the `Blackhole` to prevent the compiler optimizing and not actually running the code